docs/tutorials/elasticdl_local.md

```diff
@@ -87,11 +86,21 @@ The following commands build the Docker image `elasticdl:mnist`. Please feel
 free to name it in any other name you like.
 
 ```bash
-cd model_zoo
+cd elasticdl/model_zoo
 elasticdl zoo init
 elasticdl zoo build --image=elasticdl:mnist .
 ```
 
+## Authorize the Job Execution
+
+If you are going to run ElasticDL job in Minikube for the first time. Please
+execute the following command to authorize the execution. The command need
```

Comment:
Should be more specific to explain what this command does.

Reply:
I would suggest to

1. Merge this section into the previous section `The Kubernetes Cluster`.
1. Change the text into:
   > The following command is necessary to enable [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) of Kubernetes.

Reply:
Good suggestion, add more explaination.

Reply:
I think we can have less explanation with that URL to the explanation of RBAC.

Reply:
> I would suggest to
> 
> 1. Merge this section into the previous section `The Kubernetes Cluster`.
> 2. Change the text into:
>    > The following command is necessary to enable [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) of Kubernetes.

The elasticdl-rbac.yaml is in the elasticdl repo. We can only execute it after cloning the repo. The repo cloning work is not done in the `The Kubernetes Cluster` section yet.